OpenRA.Mods.RA/Traits/Infiltration/InfiltrateForActor.cs

```diff
@@ -13,18 +13,18 @@
 
 namespace OpenRA.Mods.RA.Traits
 {
-	class InfiltrateForSupportPowerInfo : ITraitInfo
+	class InfiltrateForActorInfo : ITraitInfo
```

Comment:
Could you clarify, please? I don't understand what you mean. I'm not completely familiar with the OpenRA jargon